### PR TITLE
add pankaj sharma card

### PR DESCRIPTION
**Pull request by Starboy-Sharma**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -112,6 +112,33 @@ <h2>A project for first-time contributions</h2>
...
+              <li>
+                <a href="https://dev.to/" target="_blank" title="Second Resource">Dev.to</a>
+              </li>
+              <li>
+                <a href="https://stackoverflow.com/" target="_blank" title="Third resource">StackOverflow</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+
```
Comment (Preet-Sojitra) [inline index.html:140]: Remove this blank space also before your "card end comment"

File: index.html [hunk 2]
```diff
@@ -112,6 +112,33 @@ <h2>A project for first-time contributions</h2>
         <!-- ============== ^^^^ TEMPLATE ^^^^ ============== -->
         <!-- ================================================ -->
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
+
+        <!-- Pankaj Sharma Card -->
+
```
Comment (Preet-Sojitra) [inline index.html:117]: Remove this blank line after your "card start comment"

**Review by Preet-Sojitra — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Starboy-Sharma) [reply index.html]: Code is updated. Plz take a look

File: index.html (hunk 2 above)
Comment (Starboy-Sharma) [reply index.html]: Code is updated. Plz take a look

**Review by nzaih1999 — APPROVED** [review]
The code is now ready for merge. Thank you for your contribution @Starboy-Sharma. Happy coding!

**Pull request merged**